docs/compilers/CSharp/Compiler Breaking Changes - DotNet 9.md

```diff
@@ -44,3 +44,26 @@ unsafe class C // unsafe context
 ```
 
 You can work around the break simply by adding the `unsafe` modifier to the local function.
+
+## .editorconfig values no longer support trailing comments
+
+***Introduced in Visual Studio 2022 version 17.[TBD]***
```

Comment:
Tagging @jaredpar to advise on which release to pull this change into, since it involves a compat break. Feels like we should hold this off rather than squeeze it late in the release cycle.

Reply:
We should either update this before merging, or open an issue to track updating it once we know which release it landed in.